pecan-waffle.psm1

```diff
@@ -1101,6 +1101,31 @@ function New-PWItem{
 }
 Set-Alias Add-Item New-PWItem -Description 'obsolete: This was added for back compat and will be removed soon'
 
+function Remove-PWTemplate{   
+    [cmdletbinding()]
+    param(
+        [Parameter(Position=1,Mandatory=$true)]
+        [ValidateNotNullOrEmpty()]
+        [string[]]$TemplateNames
+    )
+    process{
+        
+		<# Make a copy of the old template array #>
+        $templates = $Global:pecanwafflesettings.Templates
+		
+		<# Create new template array #>
+        $Global:pecanwafflesettings.Templates = @()
+
+		<# Filter out templates #>
+        foreach($template in $templates){       
+            if($TemplateNames -notcontains $template.Name){
+                $Global:pecanwafflesettings.Templates += $template
+            }
+        } 
+    }
+}
+Set-Alias Remove-Template Remove-PWTemplate 
```

Comment:
Did you add this because of the other `set-alias` elements in this file? Those are mostly there for back compat with an earlier version of the script. I think it's best to use the pattern `name-PWName` pattern. So I think we could get rid of this alias here.